src/GeneralStateTestsFiller/stEWASMTests/callValueExceeds64bitFiller.yml

```diff
@@ -0,0 +1,94 @@
+call:
```

Comment:
change the name to `callValueExceeds64bitFiller` and fill the test.  You'll get some errors ;) 

```
jwasinger@certification-hive-test03:~/projects/cpp-ethereum/build$ test/testeth -t GeneralStateTests/stEWASMTests -- --vm hera --testpath ../../tests --singletest 'callValueExceeds64bit' --filltests
Running tests using path: "../../tests"
Running 1 test case...
Test Case "stEWASMTests":
100%
/home/jwasinger/projects/cpp-ethereum/test/tools/libtesteth/TestHelper.cpp(148): error: in "GeneralStateTests/stEWASMTests": callValueExceeds64bit network not found: ALL
/home/jwasinger/projects/cpp-ethereum/test/tools/libtesteth/ImportTest.cpp(551): error: in "GeneralStateTests/stEWASMTests": callValueExceeds64bit Check State: deadbeef00000000000000000000000000000001: incorrect storage [0x38] = 0x, expected [0x38] = 0x04
Frontier data: 0 gas: 0 val: 0
/home/jwasinger/projects/cpp-ethereum/test/tools/libtesteth/TestHelper.cpp(613): error: in "GeneralStateTests/stEWASMTests": postState should not be declared in expect section!
postState:
--- 02ff5864146618008afacf94e84659304a332a719793230833d94f642565596c
     deadbeef00000000000000000000000000000000: 0 #:65535 @:56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421 $d10ca8c5a8e85b5523c36d61cb87d02bf5609d14932d2507cc3bde36d596b25e
     deadbeef00000000000000000000000000000001: 0 #:83076749736557242056487941267521535 @:56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421 $d10ca8c5a8e85b5523c36d61cb87d02bf5609d14932d2507cc3bde36d596b25e


*** 3 failures are detected (5 failures are expected) in the test module "Master Test Suite"
```